crates/client/src/client/types/block.rs

```diff
@@ -36,7 +36,7 @@ pub struct Header {
     pub transactions_count: u64,
     pub message_receipt_count: u64,
     pub transactions_root: MerkleRoot,
-    pub message_receipt_root: MerkleRoot,
+    pub message_outbox_root: MerkleRoot,
     pub height: u32,
```

Comment:
we should also include the event inbox root here, otherwise it won't be exposed to the rust-sdk or other consumers of the gql-client library.